### PR TITLE
J3rryzh/issue365

### LINKED ISSUE #365: [Finish] The animation is blocked by buttons

### LINKED ISSUE #367: Build `about us` page

### PR DESCRIPTION
**Pull request by CodeNinjaMatrix**
Fix #365 and build `about us` page

### CODE REVIEW

File: react-app/package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "lottie-react-native": "^5.1.4",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "react-native": "0.71.4",
+    "react-native": "0.71.6",
```
Comment (akaka0039) [inline react-app/package.json:26]: May I ask why did you change the version of react-native?

Comment (j3rryzh) [reply react-app/package.json:26]: It's recommended by expo. When using react-native 0.71.4, running `npx expo start` to start the preview locally, you will see the prompt.

File: react-app/src/Finish/index.js [hunk 2]
```diff
@@ -37,24 +37,26 @@ export default function Finish({ db, setReset }) {
...
         resizeMode='cover'
       />
-      <Buttons id={id} title={title} />
-    </PageTemplate>
+      <PageTemplate topBarColor={ThemeColor.background} bottomColor={ThemeColor.background}>
+        <Header />
+        <Info
+          moves={exercise.length}
+          // These coming two lines are kinda hard code
+          time={id == 1 ? 12 : 14}
```
Comment (mylesieong) [inline react-app/src/Finish/index.js:53]: We can change it to a function calculating total duration regardless of the exercise type

File: react-app/src/Mine/About.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      topBarColor={ThemeColor.background}
+      bottomColor={ThemeColor.tab}
+      removeIphoneXFooter={true}
+    >
+      <Header height={66} backgroundColor={ThemeColor.background} buttonColor={ThemeColor.text}>
+        <Text style={styles.title}>About Us</Text>
+      </Header>
+      <ScrollView style={styles.container}>
+        <Text style={styles.text}>
+          Stretch is an innovative fitness app designed to help individuals achieve their fitness
```
Comment (mylesieong) [inline react-app/src/Mine/About.js:19]: Why we break lines?

File: react-app/src/Mine/About.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+          fitness technology with expert guidance and support.
+        </Text>
+        <Text style={styles.text}>
+          We understand that everyone's fitness journey is unique, which is why we have developed a
+          range of customizable workout plans to suit every individual's needs. Our app offers
+          various options to cater to those who are just starting out on their fitness journey or
+          looking to take their workouts to the next level.
+        </Text>
+        <Text style={styles.text}>
+          At Stretch, we are passionate about helping people reach their full potential, and we
```
Comment (mylesieong) [inline react-app/src/Mine/About.js:31]: Here as well, wrong app name

File: react-app/src/Mine/About.js [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+          believe that fitness should be accessible to everyone. To that end, we have made our app
+          user-friendly and affordable so that anyone can access our innovative platform's benefits.
+          We also offer personalized coaching to our users to help them stay motivated and on track
+          towards achieving their fitness goals.
+        </Text>
+        <Text style={styles.text}>
+          We believe that fitness is not just about physical strength, but also about mental
+          wellness. That's why we have incorporated mindfulness exercises and meditation practices
```
Comment (mylesieong) [inline react-app/src/Mine/About.js:39]: We don't have mediation exercises so better remove it ;)

File: react-app/src/Mine/About.js [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+          into our app to help users manage stress and maintain a healthy balance between mind and
+          body.
+        </Text>
+        <Text style={styles.text}>
+          In conclusion, Stretch is a comprehensive fitness app that offers users access to expert
```
Comment (mylesieong) [inline react-app/src/Mine/About.js:44]: App name again

**Review by eric0039-red — APPROVED** [review]

File: react-app/src/Finish/index.js (hunk 2 above)
Comment (j3rryzh) [reply react-app/src/Finish/index.js:53]: I will create a ticket for optimizing the data structure. So I won't change this hard code for now.

File: react-app/src/Mine/About.js (hunk 3 above)
Comment (j3rryzh) [reply react-app/src/Mine/About.js:19]: For more easy reading in my opinion.

**Review by mylesieong — APPROVED** [review]
